### ISSUE TITLE
Logged in page

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-reporting#110]
#### Description:

Empty page with BCGov footer and header
Only accessible by a user who is logged in (either bceid or idir)

#### Acceptance Criteria:

Given I am a user
When I login to the reporting app
Then I land on an empty page, with the BCGov header and footer, similar to the registration app

Given I am a developer
When I want to use the bcgov header and footer components
Then they are in the shared component library

#### Development Checklist:

- [ ] Next page
- [ ] Header and footer component from the registration app, without code duplication
- [ ] Header and Footer component in the component library (maybe as an import from existing code?)
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By
 
This ticket is blocked by #108 

#### Notes:

-
